src/data/graphql/typeDefs.ts

```diff
@@ -69,7 +69,7 @@ export default gql`
     symbol: String!
     iconHash: String
     verified: Boolean!
-    balance(walletAddress: String!): String!
+    balance: String
```

Comment:
Are we sure that balance CAN be a null? I feel that here we should have String!

Reply:
yeah, probably